dotCMS/src/main/java/com/dotcms/publisher/business/PublisherAPIImpl.java

```diff
@@ -616,6 +617,7 @@ public Integer countQueueBundleIds() throws DotPublisherException {
 			dc.addParam(Status.BUNDLE_SENT_SUCCESSFULLY.getCode());
 			dc.addParam(Status.PUBLISHING_BUNDLE.getCode());
 			dc.addParam(Status.WAITING_FOR_PUBLISHING.getCode());
+			dc.addParam(new Date());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use the Java 8 Date and Time API instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2143)